src/airdrop/src/lib.rs

```diff
@@ -334,5 +336,67 @@ fn bring_caninster_back_to_life() -> CustomResult<()> {
     Ok(())
 }
 
+/// Returns all the eth addresses with how much is meant to be sent to each one of them
+#[update(guard = "caller_is_admin")]
+pub fn get_airdrop(mut index: Index) -> CustomResult<(Index, Vec<EthAddressAmount>)> {
```

Comment:
this seems like a bad pattern to mark this param as mutable

Reply:
I usually agree - however, I believe it makes sense as we give it an index and it returns a list of addresses along with the last index it stopped at, hence having one variable tracking the index might be easier for code comprehension.

Reply:
>  one variable tracking the index might be easier for code comprehension.

Well my comment proove that no. Of course it's an opiniated coding answer but, I personally would never set a params from an exposed canister update call as mutable.

Of course ultimately your call.